modules/ding_place2book/ding_place2book.module

```diff
@@ -501,7 +500,7 @@ function ding_place2book_event_op($op, $node, $place2book_id = NULL) {
 
       // Catch insert error by checking the event id.
       if (empty($req_result->headers['event-id'])) {
-        $p2b_error = $req_result->headers['status'] . ' - ' . $req_result->status_message;
```

Comment:
These changes regarding error messages does not seem to be related to the change at hand?

Also: I could not find any documentation of these `error-message` and `error-class` headers anywhere. If we want to use them is there and url we could refer to or the like? A inline comment explaining the situation could also be helpful.

Reply:
Likewise there was no documentation for the `status`. Either way this gotta be consulted with place2book api. The scope was to fix the notices.